addons/account/views/partner_view.xml

```diff
@@ -283,4 +283,4 @@
             </field>
         </record>
     </data>
-</odoo>
+</odoo>
```

Comment:
useless diff here

Reply:
I agree but those two are strictly identical... ? Why is it a diff?

Reply:
You (or your editor) removed the end of line character.
![image](https://user-images.githubusercontent.com/44770049/144572798-5f30163e-d92c-4899-b848-142e5431b440.png)